src/go/configinfo/method_info.go

```diff
@@ -40,6 +40,10 @@ type methodInfo struct {
 
 	// The request type name (not the entire type URL).
 	RequestTypeName string
+
+	// The auto-generated methods, used to replace snakeName with jsonName in their
```

Comment:
Does this need to be a slice? There should only be 1 method generated for each one.